Source/WebCore/page/LocalDOMWindow.cpp

```diff
@@ -2763,7 +2768,7 @@ void LocalDOMWindow::disableSuddenTermination()
 
 LocalFrame* LocalDOMWindow::frame() const
 {
-    auto* document = this->document();
+    SUPPRESS_UNCOUNTED_LOCAL auto* document = this->document();
```

Comment:
Please do not add SUPPRESS_UNCOUNTED_LOCAL.

Reply:
Removed.